gh-pages/src/baseline-in-the-wild.md

```diff
@@ -40,6 +40,7 @@ You can find web-features and Baseline data in the places listed below. If you s
 - [Baseline web features for the win](https://www.w3.org/2024/09/TPAC/demo-baseline.html) by Patrick Brosset, TPAC, August 2024.
 - [Baseline features and webstatus.dev](https://www.youtube.com/watch?v=pTsMpoXGlqE) ([slides](https://docs.google.com/presentation/d/1dRWC7aH-FQTj2JVFIaRvrHNylaRKAUCnQ5Y4odRhGGY/edit#slide=id.g2f87bb2d5eb_0_4)) by Kadir Topal and James Scott, BlinkOn, October 2024.
 - [Baseline and Web Features](https://www.oddbird.net/2024/11/19/winging-it-13/) by Miriam Suzanne and James Stuckey Weber, Winging It Live, November 2024.
+- [web-features and Baseline](https://www.youtube.com/watch?v=QzfwNFIXOkM) by Patrick Brosset, W3C AC meeting, April 2025.
```

Comment:
Optional suggestion; feel free to merge with or without, whenever you're ready.

I'm happy to see this added, but now that there are several of these I'm seeing a minor problem: we're putting the more recent (and probably most interesting?) presentations last. Would you mind reversing the order of this list?